include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -35,7 +35,10 @@ using void_type = typename ::std::enable_if<::std::is_void<_Tp>::value, _Tp>::ty
 template <typename _Tp>
 using non_void_type = typename ::std::enable_if<!::std::is_void<_Tp>::value, _Tp>::type;
 
-#if _USE_GROUP_ALGOS
+#if _USE_GROUP_ALGOS && defined(__INTEL_LLVM_COMPILER)
```

Comment:
From my point of view we need to add `#define _ONEDPL_SYCL_INTEL_COMPILER` in our config that should be defined in `1` only `#if defined(__INTEL_LLVM_COMPILER) and defined(SYCL_LANGUAGE_VERSION)` and then use our macro here for checking.

Reply:
Good point. Thank you for comment, I have applied it.